dev/README.md

```diff
@@ -1,27 +1,72 @@
 # Development setup
 
-This directory contains the docker compose file to run an isolated environment of the full stack during development.
+This directory contains scripts to set up, run, and test the full stack during development, including:
 
-## Prerequisite
+- `run_stack.sh`: A script to run the dev docker compose file that runs an isolated environment of the full stack.
+- `setup-db.sh`: A script to that connects to your locally-running MongoDB instance and (re-)creates the database and
+  collection, applies the schema, and inserts some sample data.
+- `test_all.sh`: A script to run all of the tests from the sub-packages.
 
-**Important** running the full stack correctly requires having access to a few secrets: you should have a .env file at the top of your repository (where you run the `docker-compose` command) that looks like this:
+What, exactly, is this stack that we're running and/or testing? All of the components of this repo working together: a
+curator UI server that talks to a curator API server that talks to a data server. More concretely, a user can log into
+the curator UI to manage case report data sources and view or update case data, with more functionality coming soon.
 
-```
+## Getting up and running
+
+### Prerequisites
+
+#### Software
+
+- [Homebrew](https://docs.brew.sh/Installation): To facilitate installing the below.
+- [MongoDB](https://docs.mongodb.com/manual/administration/install-community/): Make sure, after install, you can run
```

Comment:
nit. in practice we're using the docker image of mongo so you don't need to install mongodb (not a *prerequisite*), but the CLI tools are nice to have (mongoimport and others

Reply:
sadly they are needed for setup_db script :(